docker-compose.yml

```diff
@@ -23,10 +23,12 @@ services:
     image: ghcr.io/ndph-arts/trial-config-service:${GITHUB_SHA}
     depends_on:
       - sqlserver
+      - role-service
     environment:
       SERVER_PORT: "81"
       JDBC_DRIVER: ${JDBC_DRIVER}
       JDBC_URL: ${JDBC_URL}
+      ROLE_SERVICE: "http://role-service-container:82"
```

Comment:
Shouldn't this just be the service name, rather than container? I.e. role-service:82

Reply:
Thanks Mark, good point. I have changed this.